### PR TITLE
Update README_EN.md

### PR DESCRIPTION
**Pull request by Bernhard-L**
C++20 solution using std::ranges::unique from algorithm

### CODE REVIEW

File: solution/0000-0099/0026.Remove Duplicates from Sorted Array/README_EN.md [hunk 1]
```diff
@@ -91,11 +91,9 @@ class Solution {
 class Solution {
 public:
     int removeDuplicates(vector<int>& nums) {
-        int i = 0;
-        for (int& num : nums)
-            if (i < 1 || num != nums[i - 1])
-                nums[i++] = num;
-        return i;
+        auto ret = std::ranges::unique(nums);
+        nums.erase(ret.begin(), ret.end());
```
Comment (poyea) [inline solution/0000-0099/0026.Remove Duplicates from Sorted Array/README_EN.md:95]: Note that this is equivalent to, before C++20,
```cpp
nums.erase(unique(nums.begin(), nums.end()), nums.end());
```

**Review by poyea — COMMENTED** [review]
Do we want to replace this implementation?

File: solution/0000-0099/0026.Remove Duplicates from Sorted Array/README_EN.md (hunk 1 above)
Comment (yanglbme) [reply solution/0000-0099/0026.Remove Duplicates from Sorted Array/README_EN.md:95]: We can add a new C++ implenentation.

### **Java**

...

### **C++**

```cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0;
        for (int& num : nums)
            if (i < 1 || num != nums[i - 1])
                nums[i++] = num;
        return i;
    }
};
```

```cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        nums.erase(unique(nums.begin(), nums.end()), nums.end());
        return nums.size();
    }
};
```

### **Go**

...

Comment (yanglbme) [reply solution/0000-0099/0026.Remove Duplicates from Sorted Array/README_EN.md:95]: And this implementation should be added to both README.md and README_EN.md files.

Comment (yanglbme) [reply solution/0000-0099/0026.Remove Duplicates from Sorted Array/README_EN.md:95]: @Bernhard-L

Comment (Bernhard-L) [reply solution/0000-0099/0026.Remove Duplicates from Sorted Array/README_EN.md:95]: fine by me

Comment (poyea) [reply solution/0000-0099/0026.Remove Duplicates from Sorted Array/README_EN.md]: @Bernhard-L This was just a comment. I'd also love C++20 features here too! Sorry for this

**Review by yanglbme — APPROVED** [review]

**Pull request merged**